lib/codegen/model.ejs

```diff
@@ -0,0 +1,87 @@
+<%
+function printParams() {
+    var params = [];
+    for (var j = 0; j < op.accepts.length; j++) {
+        var param = op.accepts[j];
+        var type = param.type || '*';
+        if (Array.isArray(param.type)) {
+            type = param.type[0] + '[]';
+        }
+        params.push('* @param {' + type + '} ' + param.arg + ' ' + param.description);
+    }
+    return params.join('\n');
+}
+
+
+function printMethod(modelName) {
+    var className = modelName || 'Model';
+    var method = className + '.' + op.operationId
+            + ' = function(';
+    var params = op.accepts.map(function(a) {
+        return a.arg;
+    });
+    params.push('callback');
+    return method + params.join(', ') + ') {';
+}
+
+function printRequestArgsOld() {
+    var requestArgs;
+    for (var j = 0; j < op.accepts.length; j++) {
+        var param = op.accepts[j];
+        if (j != 0) {
+            requestArgs = requestArgs + ",";
+        }
+        requestArgs = param.arg + ": " + param.arg;
+    }
+    requestArgs = "{\n" + "             " + requestArgs + "\n        }";
```

Comment:
Since this module required node v4 or newer you might want to consider using the new [template string literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) that were introduced in ES2015. In this case you would probably only be using them for simple multi-line strings with interpolation support:
```js
requestArgs = `{
             ${requestArgs}
        }`;
```